src/main/java/com/conveyal/data/census/S3SeamlessSource.java

```diff
@@ -13,12 +13,23 @@
  * A seamless data source based on storage in Amazon S3.
  */
 public class S3SeamlessSource extends SeamlessSource {
-    private static AmazonS3 s3 = AmazonS3ClientBuilder.defaultClient();
+    private static AmazonS3 s3;
 
+    public final String region;
     public final String bucketName;
 
     public S3SeamlessSource(String bucketName) {
+        this.region = null;
         this.bucketName = bucketName;
+        this.s3 = AmazonS3ClientBuilder.defaultClient();
+    }
+
+    public S3SeamlessSource(String region, String bucketName) {
```

Comment:
Would the best option just be to pass in an `AmazonS3` instance and the bucket name? That way it can be shared.